src/main/java/models/Game.java

```diff
@@ -116,41 +53,19 @@ public void customDeal(int c1, int c2, int c3, int c4) {
         deck.remove(c4);
     }
 
-    public void remove(int columnNumber) {
-        if (colHasCards(columnNumber)) {
-            Card c = getTopCard(columnNumber);
-            Boolean removeCard = false;
-            for (int i = 0; i < 4; i++) {
-                if (i != columnNumber) {
-                    if (colHasCards(i)) {
-                        Card compare = getTopCard(i);
-                        if (compare.getSuit() == c.getSuit()) {
-                            if (compare.getValue() > c.getValue()) {
-                                removeCard = true;
-                            }
-                        }
-                    }
-                }
-            }
-            if (removeCard) {
-                this.cols.get(columnNumber).remove(this.cols.get(columnNumber).size() - 1);
-                this.errMsg = "No Error";
-            }
-            if (!removeCard) {
-                this.errMsg = "Invalid remove!";
-            }
-        }
+    public abstract void remove(int columnNumber);
+
+
 
-    }
 
-    private boolean colHasCards(int colNumber) {
+    protected boolean colHasCards(int colNumber) {
         if(this.cols.get(colNumber).size()>0){
             return true;
         }
         return false;
     }
 
-    private Card getTopCard(int columnNumber) {
+    protected Card getTopCard(int columnNumber) {
```

Comment:
I changed them to protected in order for the child game classes to be able to access them while still providing a level of security that Public does not offer.